commands/scan/buildscan.go

```diff
@@ -155,7 +155,6 @@ func (bsc *BuildScanCommand) runBuildScanAndPrintResults(xrayManager *xray.XrayS
 
 	resultsPrinter := utils.NewResultsWriter(scanResults).
 		SetOutputFormat(bsc.outputFormat).
-		SetIncludeVulnerabilities(bsc.includeVulnerabilities).
 		SetIncludeLicenses(false).
```

Comment:
after removing this line - did you test the CLI behaviour with and without watches?

Reply:
@hadarshjfrog I did the regular tests (also with watches).
This line doesn't affect watches as it relates to vulnerabilities. (and watches are affecting violations).
We call SetIncludeVulnerabilities in line 179 if vulnerabilities should be added:
`		if bsc.includeVulnerabilities {
			resultsPrinter.**SetIncludeVulnerabilities**(true)
			if err = resultsPrinter.PrintScanResults(); err != nil {
				return false, err
			}
		}`